src/containers/StructurePage/folderComponents/menuOptions/AddExistingTopic.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Plus } from 'ndla-icons/action';
+import Button from 'ndla-button';
+import InlineDropdown from '../../../../components/Dropdown/InlineDropdown';
+import RoundIcon from '../../../../components/RoundIcon';
+import { fetchTopics, addTopicToTopic } from '../../../../modules/taxonomy';
+
+class AddExistingTopic extends React.PureComponent {
+  constructor() {
+    super();
+    this.onAddExistingSubTopic = this.onAddExistingSubTopic.bind(this);
+    this.toggleEditMode = this.toggleEditMode.bind(this);
+    this.fetchTopicsLocale = this.fetchTopicsLocale.bind(this);
+  }
+
+  async onAddExistingSubTopic(subTopicId) {
+    const { id, numberOfSubtopics, refreshTopics } = this.props;
+    await addTopicToTopic({
+      subtopicid: subTopicId,
+      topicid: id,
+      primary: false,
+      rank: numberOfSubtopics + 1,
+    });
+    refreshTopics();
+  }
+
+  toggleEditMode() {
```

Comment:
Er vel ikke helt nødvendig å ha denne funksjonen?

Reply:
Nei, gjorde det mest for å gjøre render mer lesbar, samt at det er bittelitt raskere. Kan ta det vekk, evt kan vi åpne for stage 3 og slippe bind overalt?